docs/pages/router/advanced/nesting-navigators.mdx

```diff
@@ -24,14 +24,14 @@ Consider the following file structure which is used as an example in this guide:
   ]}
 />
 
-In the above example, **app/home/feed.js** matches `/home/feed` and **app/home/messages.js** matches `/home/messages`.
+In the example above, **app/home/feed.js** matches `/home/feed` and **app/home/messages.js** matches `/home/messages`.
```

Comment:
```suggestion
In the above example, **app/home/feed.js** matches `/home/feed`, and **app/home/messages.js** matches `/home/messages`.
```

- Not a significant change; we follow the same pattern elsewhere in the docs.
- Added missing oxford comma (,)